src/dnx.coreclr.unix/dnx.coreclr.cpp

```diff
@@ -124,13 +148,30 @@ void* LoadCoreClr(std::string& runtimeDirectory)
         runtimeDirectory.erase(lastSlash);
```

Comment:
Yes.  The caller of this function needs to know the directory the runtime was loaded from so it can use it later.